src/parsers/installrdf.js

```diff
@@ -3,6 +3,9 @@ import * as messages from 'messages';
 import log from 'logger';
 import { singleLineString } from 'utils';
 
+const TARGET_APPLICATION_FIELDS = [
```

Comment:
Maybe move to `const` and add something to the name to reflect this is to do with RDF?